src/lib/css/index.scss

```diff
@@ -1,32 +1,23 @@
-@use "sass:map";
+@layer theme, ripple, components, classes, disabled;
 
-@import "./theme/theme";
-@import "classes";
-@import "ripple";
+@layer Quaff {
+  @layer theme {
+    @import "./theme/theme";
+  }
 
-@import "$components/checkbox";
-@import "$components/drawer";
-@import "$components/footer";
-@import "$components/input";
-@import "$components/layout";
-@import "$components/radio";
-@import "$components/railbar";
-@import "$components/select";
-@import "$components/table";
-@import "$components/toolbar";
-@import "$components/tooltip";
+  @layer ripple {
+    @import "ripple";
+  }
 
-html,
-body {
-  height: 100%;
-  width: 100%;
-  color: var(--on-surface);
-}
+  @layer components {
+    @import "components";
```

Comment:
Tree shakability might not work here, but it's something we don't need to consider for this PR.